Unity-Technologies/ScriptableRenderPipeline
# LW Handle final blit in SRP

### Purpose of this PR

Some platforms (Android) are doing a system blit when the backbuffer doesn't support a certain configuration. This PR makes LW responsible for doing this blit, removing double-blit situations.

---
### Release Notes

LWRP now handles system bliss for platforms and configurations that require this, removing a redundant blit operation.

---
### Testing status
**Katana Tests**: https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?automation-tools_branch=add-platform-filter&ScriptableRenderLoop_branch=lw%2Fhandle_final_blit_in_srp&unity_branch=trunk

**Manual Tests**: What did you do?
Manually tested on a device (one for now, will test some later).

**Automated Tests**: What did you setup?
Nothing!

Any test projects to go with this to help reviewers?
Nope, just use the template, if you want to check this :)
---
### Overall Product Risks
**Technical Risk**: None, Low, Medium, High?
Low.

**Halo Effect**: None, Low, Medium, High?
Low (Android and iOS).
---
### Comments to reviewers
Notes for the reviewers you have assigned.
